app/jobs/notify_sms_delivery_job.rb

```diff
@@ -3,6 +3,8 @@
 class NotifySmsDeliveryJob < ApplicationJob
   queue_as :mailers
 
+  sidekiq_options retry: false
```

Comment:
You could put that in `ApplicationJob`

Reply:
I'm not sure we want this behaviour for all jobs, especially those where we haven't set any retry preferences (e.g. the Zendesk ticket worker).

Reply:
We could avoid mixing two different retry logics with something like

```ruby
  sidekiq_options retry: false
  retry_on StandardError, wait: :exponentially_longer, attempts: 5
```

Reply:
That seems like a good idea.  I'll update this PR.